translations/en.json

```diff
@@ -1106,7 +1106,7 @@
     "wallet-manage-assets": "Manage assets",
     "wallet-offline": "Wallet offline",
     "wallet-request": "Request",
-    "wallet-send": "Send",
+    "wallet-send": "Next",
```

Comment:
this is not only going to affect the signing flow, this is also going to affect the send button in the wallet UI. The correct way to change this would rather be to use the "next" label in the bottom sheet instead of changing the meaning of this one